tgui/packages/common/storage.ts

```diff
@@ -6,99 +6,128 @@
  * @license MIT
  */
 
+declare interface Window {
```

Comment:
useless, global.d.ts already defines this

Reply:
interfaces in typescript have a weird property of merging all interfaces with the same name into one single interface, but I agree this should be in global.d.ts